blog/content/post/2020-04-16-remote_work_post.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Remote productivity
+abstract: 'About remote productivity in 3mdeb.'
+cover: /covers/remote-work.jpg
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-04-16
+archives: "2020"
+
+tags:
+  - 3mdeb team
+  - productivity
+  - aboutUs
+categories:
+  - Miscellaneous
+---
+### How remote working affects productivity
+
+Since the recent COVID-19 outbreak we all have been forced to pack our tools
+along with our potential and switch to home office. One may say that for IT it
+is not a milestone; after all our work didn’t change significantly. We can still
+develop solutions of an invariably good quality providing that our remote
+environment has been fully set. Apart from tools there are more factors,
+combined in workflow, of which we are deprived while working remotely for a long
+term. Among endless interruptions and no watchful eyes around, with difficulties
+to connect people directly, there is the question that rises inside the head:
+<b> Ping or not to ping? </b>
```

Comment:
@KBanecka html elements are necessary evil in this case - please use markdown emphasis characters:

\*\*bold\*\* -> **bold**

or

\_\_bold\_\_ -> __bold__

Reply:
changed

Reply:
done